tests/avatar/manager_test.php

```diff
@@ -99,7 +99,7 @@ public function setUp()
 		$this->config['allow_avatar_' . get_class($this->avatar_barfoo)] = false;
 
 		// Set up avatar manager
-		$this->manager = new \phpbb\avatar\manager($this->config, $avatar_drivers, $phpbb_container);
+		$this->manager = new \phpbb\avatar\manager($this->config, $dispatcher, $avatar_drivers, $phpbb_container);
```

Comment:
Erm, why is the container still being passed? Don't think that's needed anymore ;)